qutip/cy/pyxbuilder.py

```diff
@@ -32,9 +32,19 @@
 ###############################################################################
 import sys
 import os
+import warnings
 
 import numpy as np
-import pyximport
+
+with warnings.catch_warnings():
+    # pyximport loads the imp module, which is deprecated in favour of
```

Comment:
Sometimes I find it useful to start such comments with "TODO:" so that one can easily search the code for things we wanted to sort out but might have forgotten about.